src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html

```diff
@@ -5,6 +5,12 @@
     Submit Feedback
   </h1>
   <br>
+  <div id="student-motd-wrapper">
+    <input id="motd-url" type="hidden" value="${studentmotd.url}">
+    <script defer="" src="/js/studentMotd.js" type="text/javascript">
+    </script>
+    ${studentmotd.container}
+  </div>
```

Comment:
We are currently investigating this now. This has popped up quite a few times recently and we have no idea why.